File: debug_sections.py
from supabase import create_client
import os
from dotenv import load_dotenv

load_dotenv('.env.local')

url = os.environ.get("NEXT_PUBLIC_SUPABASE_URL")
key = os.environ.get("NEXT_PUBLIC_SUPABASE_ANON_KEY")
supabase = create_client(url, key)

res = supabase.table('courses').select('id', 'title', 'sections').execute()

for course in res.data:
    print(f"Course: {course['id']} - {course['title']}")
    sections = course.get('sections', [])
    if not sections:
        print("  No sections")
        continue
    for sec in sections:
        if sec.get('type') == 'detailed_features' or 'Features' in str(sec.get('title', '')):
            print(f"  Section Type: {sec.get('type')} | Badge: {sec.get('badge')} | Title: {sec.get('title')}")

File: debug_sections.js
const { createClient } = require('@supabase/supabase-js');
require('dotenv').config({ path: '.env.local' });

const supabase = createClient(
    process.env.NEXT_PUBLIC_SUPABASE_URL,
    process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY
);

async function check() {
    const { data: courses, error } = await supabase
        .from('courses')
        .select('id', 'title', 'sections');

    if (error) {
        console.error(error);
        return;
    }

    courses.forEach(course => {
        console.log(`Course: ${course.id} - ${course.title}`);
        const sections = course.sections || [];
        if (sections.length === 0) {
            console.log("  No sections");
            return;
        }
        sections.forEach(sec => {
            if (sec.type === 'detailed_features' || (sec.title && sec.title.includes('Features'))) {
                console.log(`  Section Type: ${sec.type} | Badge: ${sec.badge} | Title: ${sec.title}`);
            }
        });
    });
}

check();
